app/Http/Services/WithdrawalService.php

```
@@ -0,0 +1,99 @@
+                    'item_id'        => $item['item_id'],
+                    'quantity'       => $totalDeducted,
+                    'uom_id'         => $deductions[0]['uom_id'] ?? null,
+                    'uom_conversion' => $deductions[0]['uom_conversion'] ?? null,
+                    'metadata'       => [
+                        'reason'                => 'Withdrawal',
+                        'request_withdrawal_id' => $item['request_withdrawal_id'],
+                        'deductions'            => $deductions,
+                    ],
+                ]);
+            }
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'withdrawItemsFromWarehouse'. (undefined)

(StaticAccess)